manager/commands/genscen.py

```diff
@@ -9,6 +9,8 @@
     "name": "template",
     "rounds": 0,
     "blocks": 120,
+    "default_version": "2.0.4",
+    "distributor_version": "2.0.4",
```

Comment:
I'd prefer not adding these lines so that the default is set only in `manager.py`.
```suggestion
```